apps/web/src/components/templates/TemplatePageHeader.tsx

```diff
@@ -0,0 +1,94 @@
+import { Center, Container, Grid, Group } from '@mantine/core';
+import { useState } from 'react';
+import { Button, colors, Switch, Title, Text } from '../../design-system';
+import { ArrowLeft } from '../../design-system/icons';
+import { useEnvController } from '../../store/use-env-controller';
+import { When } from '../utils/When';
+import { EditorPreviewSwitch } from './EditorPreviewSwitch';
+import { useStatusChangeControllerHook } from './use-status-change-controller.hook';
+import { useTemplateController } from './use-template-controller.hook';
+
+const Header = ({ activePage, editMode }) => {
+  if (activePage === 'Settings') {
```

Comment:
A small suggestion. is it possible to have this input an Enum to use across the board? I think it can be easier to deal with.

Reply:
Memory is to short, me and @ainouzgali spoke about this when I did this issue and I forgot about it...